.CI/create_feedstocks.py

```diff
@@ -186,9 +186,26 @@ def write_token(name, token):
 
             subprocess.check_call(['conda', 'smithy', 'rerender'], cwd=feedstock_dir)
             subprocess.check_call(['git', 'commit', '-am', "Re-render the feedstock after CI registration."], cwd=feedstock_dir)
-            # Capture the output, as it may contain the GH_TOKEN.
-            out = subprocess.check_output(['git', 'push', 'upstream_with_token', 'master'], cwd=feedstock_dir,
-                                          stderr=subprocess.STDOUT)
+            not_pushed_feedstock = True
+            while not_pushed_feedstock:
+                try:
+                    # Capture the output, as it may contain the GH_TOKEN.
+                    out = subprocess.check_output(['git', 'push', 'upstream_with_token', 'master'], cwd=feedstock_dir,
+                                              stderr=subprocess.STDOUT)
+                    not_pushed_feedstock = False
+                except subprocess.CalledProcessError:
+                    not_pushed_feedstock = True
+
+                if not_pushed_feedstock:
+                    # Likely another job has already pushed to this repo.
+                    # Place our changes on top of theirs and try again.
+                    subprocess.check_call(['git', 'fetch', 'upstream_with_token', 'master'], cwd=feedstock_dir)
+                    try:
+                        subprocess.check_call(['git', 'rebase', 'upstream_with_token/master', 'master'], cwd=feedstock_dir)
```

Comment:
So I don't think this is an issue. Rebase doesn't actually printing anything about the branches involved. It also obviously doesn't do a fetch. Even if there is a conflict, it just notes the files in conflict and asks that they be resolved.